### ISSUE TITLE
it doesnt work in Ableton 11

### ISSUE THREAD

**Issue by ish-** [wiffbi/Selected_Track_Control#6]
i think its because they did update python engine
would you check it?

**Comment by wiffbi** [comment]
Yes, the upgrade to Python 3. I send you a beta with Python 3 support with input/output routing and quantization settings not working. I will look into this and try to update STC. I do not know when though.

**Comment by frnktrpz** [comment]
Is the beta available anywhere? Feel kinda lost without STC and could live with the routing and quantise controls not working for now :-) 

Looked into converting the scripts to python 3 myself but soon came to the conclusion that it was way over my head.

**Comment by wiffbi** [comment]
Yes, I updated the homepage http://stc.wiffbi.com/ with a link to the beta. Or get it directly here: http://stc.wiffbi.com/files/stc-v1.4.0-beta.zip

**Comment by frnktrpz** [comment]
Awesome, thanks!

**Comment by ish-** [comment]
thank you so much!!
